src/components/Create/Create.jsx

```diff
@@ -81,42 +153,93 @@ function Create(props) {
                 ...prevState,
                 name: e.value
             }))
-            setWarningText(null)
+            setNameText(null)
         }
     } 
 
     // handle create catchment
     const handleCreate = async () => {
-        if (formInputs.name === "" ) return
-        if (formInputs.country === "") return
-        // TODO: to create catchment, will need data from DHIS2
-        // {
-        //     name,
-        //     csv,
-        //     country,
-        //     algorithm,
-        //     fields: {
-        //         lat,
-        //         lng,
-        //         name
-        //     }
-        // }
-        await createCatchmentJob(formInputs)
+        if (formInputs.country === "") {
+            setCountryText(i18n.t("Country required"))
+            return
+        }
+        if (formInputs.name === "") {
+            setNameText(i18n.t("Name required"))
+            return
+        }
+        if (formInputs.level === "") {
+            setLevelText(i18n.t("Level required"))
+            return
+        }
+        
+        setIsLoading(true)
+        const resp = await createCatchmentJob(formInputs).catch( async (err) => {
+            setIsLoading(false)
+            const data = JSON.parse(await err.response.text())
+
+            if (data.csv) {
+                const resp = papaparse.parse(data.csv.trim(), { header: true })
+                return { error: resp }
+            } else {
+                return setAlertError({ text: i18n.t(data.message), critical: true })
+            }
+        })
+        
+        // if there are errors then set the error
+        if (resp?.error) {
+            const errors = resp.error.data.filter((data) => data["cc:ErrorMessage"] !== "") 
+            const clean = resp.error.data.filter((data) => data["cc:ErrorMessage"] === "")
```

Comment:
If it's easy to modify, I think it would be better to show the user a column with the title "ERROR MESSAGE" instead of "cc:ErrorMessage" -- having the "cc:" makes it look out of place with the other column headings.